cli/cmd/compose/up.go

```diff
@@ -74,7 +79,9 @@ func upCommand(p *projectOptions, contextType string) *cobra.Command {
 	if contextType == store.AciContextType {
 		flags.StringVar(&opts.DomainName, "domainname", "", "Container NIS domain name")
 	}
-
+	if contextType == store.LocalContextType {
+		flags.StringVar(&opts.PullPolicy, "pull", "", "Define pull policy for service images (always|never|if_not_present|build).")
```

Comment:
`docker run` uses "missing" for the `if_not_present` case (which is the default);

```
      --pull string                    Pull image before running ("always"|"missing"|"never") (default "missing")
```

For backward-compatibility (`--pull` used to be a "boolean" on `docker run`), specifying `--pull` without a value is equivalent to `--pull=always`. Not sure if we need to add that logic here (because this is a new flag).

What's the `build` option? It's a bit unclear to me what it's intended for; if that's meant for "only do a pull if we also have to do a build", should we keep it simple, and keep that for `docker compose build [--pull=<policy>]` ?

Reply:
this is just plain https://github.com/compose-spec/compose-spec/blob/master/spec.md#pull_policy

Reply:
that's a bit unfortunate defining this pull_policy as part of the compose-spec was not coordinated with introducion of the docker run `pull` flag for consistency :'(

Reply:
With the compose-spec not having a release yet, could we still change? I see I left a comment at the time in https://github.com/compose-spec/compose-spec/pull/78#discussion_r435800728 as well

TBH; Overall a bit on the fence of having it as an option in the compose-file itself, as it feels more of an option that should be set by the person doing the deploy, not a "declarative" option that specifies/defines the stack